lib/maze/timers.rb

```diff
@@ -0,0 +1,52 @@
+module Maze
+
+  # A simple run/stop timer
+  class Timer
+    attr_accessor :total
+
+    def initialize
+      @started = 0
```

Comment:
Can this be removed, it doesn't seem to be used anywhere

Reply:
Yes, you're quite right.